### PR TITLE
chore: unskip PG tests against the emulator

### PR DESCRIPTION
**Pull request by hengfengli**
Unskip PG tests against the emulator which are disabled previously. Note that this needs to run against the latest emulator which includes FLOAT32 support.

### CODE REVIEW

File: google-cloud-spanner/acceptance/spanner_helper.rb [hunk 1]
```diff
@@ -140,34 +140,32 @@ def assert_commit_response resp, commit_options = {}
 db_job.wait_until_done!
 raise GRPC::BadStatus.new(db_job.error.code, db_job.error.message) if db_job.error?
 
-unless emulator_enabled?
-  instance_path = $spanner_db_admin.instance_path project: $spanner.project_id, instance: $spanner_instance_id
-  db_job = $spanner_db_admin.create_database parent: instance_path,
+instance_path = $spanner_db_admin.instance_path project: $spanner.project_id, instance: $spanner_instance_id
+db_job = $spanner_db_admin.create_database parent: instance_path,
```
Comment (dazuma) [inline google-cloud-spanner/acceptance/spanner_helper.rb:144]: Please adjust the indentation of the following two lines to match the change to this line. (Corrects rubocop failure.)

File: google-cloud-spanner/acceptance/spanner_helper.rb [hunk 2]
```diff
@@ -140,34 +140,32 @@ def assert_commit_response resp, commit_options = {}
...
                                              create_statement: "CREATE DATABASE \"#{$spanner_pg_database_id}\"",
                                              database_dialect: :POSTGRESQL
-  db_job.wait_until_done!
-  raise GRPC::BadStatus.new(db_job.error.code, db_job.error.message) if db_job.error?
-  db_path = $spanner_db_admin.database_path project: $spanner.project_id,
+db_job.wait_until_done!
+raise GRPC::BadStatus.new(db_job.error.code, db_job.error.message) if db_job.error?
+db_path = $spanner_db_admin.database_path project: $spanner.project_id,
```
Comment (dazuma) [inline google-cloud-spanner/acceptance/spanner_helper.rb:149]: Please adjust the indentation of the following two lines to match the change to this line. (Corrects rubocop failure.)

**Review by dazuma — CHANGES_REQUESTED** [review]

File: google-cloud-spanner/acceptance/spanner_helper.rb (hunk 2 above)
Comment (hengfengli) [reply google-cloud-spanner/acceptance/spanner_helper.rb:149]: Fixed.

File: google-cloud-spanner/acceptance/spanner_helper.rb (hunk 1 above)
Comment (hengfengli) [reply google-cloud-spanner/acceptance/spanner_helper.rb:144]: Fixed.

**Pull request merged**